packages/contracts/test/deploy/managing-dao.ts

```diff
@@ -42,9 +48,11 @@ describe('Managing DAO', function () {
   let ensSubdomainRegistrarDeployments: Deployment[];
   let ensSubdomainRegistrars: ENSSubdomainRegistrar[];
 
-  async function createUpgradeProposal(
+  async function createAndExecuteUpgradeProposal(
     contractAddress: string[],
-    newImplementationAddress: string
+    newImplementationAddress: string,
+    approvers: SignerWithAddress[],
+    minApprovals: number
```

Comment:
perhaps we can use `approvals` instead of `minApprovals`
or we can make this function more generic, and pass the proposal to it...
WDYT?

Reply:
See https://github.com/aragon/osx/pull/440/files#r1286994800